elasticdl/python/tests/worker_ps_interaction_test.py

```diff
@@ -266,6 +270,67 @@ def test_worker_pull_embedding(self):
             expected_result = np.concatenate(expected_result)
             self.assertTrue(np.allclose(expected_result, result_dict[layer]))
 
+    def test_deepfm_train(self):
```

Comment:
Extract the common training logic of `test_deepfm_train` and `test_mnist_train` as a function.